### ISSUE TITLE
base description tag not being set

### ISSUE THREAD

**Issue by yogeshVU** [webgme/webgme-taxonomy#395]
although the display tag gets populated and hence can be viewed on the dashboard, the 'description' tagging for the base is not being currently set.

**Comment by brollb** [comment]
yeah, there isn't a place to set it currently. Perhaps it shouldn't be a system term but should be a regular term in the `Base` vocabulary...

**Comment by yogeshVU** [comment]
it would be good to revisit this issue, if we are still setting the description term.
